### PR TITLE
Distinguish TLS verification flag

### LINKED ISSUE #15: Distinguish TLS verification flag between internal and external registry
**Is your feature request related to a problem? Please describe.**

As of now, there is no flag to distinguish TLS verification from internal and external registry - there is used just one flag that controls both TLS verifications.

**Describe the solution you'd like**

Remove `--no-registry-tls-verify` and introduce `--no-src-registry-tls-verify`, `--no-dst-registry-tls-verify` flags.

**Additional context**

https://github.com/thoth-station/build-watcher/blob/475620418530286f44db1aa75538ee4c6cf005a9/app.py#L190-L191

### PR DESCRIPTION
**Pull request by bissenbay**
Fixes https://github.com/thoth-station/build-watcher/issues/15

### CODE REVIEW

**Comment by goern** [comment]
@bissenbay  still WIP?

**Comment by bissenbay** [comment]
> @bissenbay still WIP?

Deleted WIP label

File: app.py [hunk 1]
```diff
@@ -332,12 +333,18 @@ def _submitter(
...
-    "--no-registry-tls-verify",
-    "-R",
+    "--no-src-registry-tls-verify",
+    "-S",
     is_flag=True,
-    envvar="THOTH_NO_REGISTRY_TLS_VERIFY",
-    help="Do not check for TLS certificates of registry when pulling images on Thoth side or "
-    "when pushing to a remote push registry.",
+    envvar="THOTH_NO_SOURCE_REGISTRY_TLS_VERIFY",
+    help="Do not check for TLS certificates of registry when pulling images on Thoth side.",
```
Comment (bissenbay) [inline app.py:340]: Does `help` description look good?

File: app.py [hunk 2]
```diff
@@ -332,12 +333,18 @@ def _submitter(
...
+)
+@click.option(
+    "--no-dst-registry-tls-verify",
+    "-D",
+    is_flag=True,
+    envvar="THOTH_NO_DESTINATION_REGISTRY_TLS_VERIFY",
+    help="Do not check for TLS certificates of registry when pushing to a remote push registry.",
```
Comment (bissenbay) [inline app.py:347]: Looks good?

**Review by fridex — COMMENTED** [review]
Code changes LGTM. Could you also adjust deployment - openshift templates and ansible playbooks (with this change the deployment will not propagate parameters correctly)?

**Review by goern — COMMENTED** [review]
LGTM

**Review by fridex — APPROVED** [review]
LGTM, thanks!

**Pull request merged**